example-policies/example-policy1.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: delivery.knative.dev/v1alpha1
```

Comment:
I think it makes sense to have 2 policies:

1. basic.yaml (this is pretty much policy2)
2. advanced.yaml

basic only uses default threshold. advanced uses increasing thresholds.

Reply:
fixed